### PR TITLE
Kajam 11.3.0 plat 3963

### PR DESCRIPTION
**Pull request by hilak**

### CODE REVIEW

File: plugins/content_distribution/providers/comcast_mrss/lib/ComcastMrssFeed.php [hunk 1]
```diff
@@ -300,6 +313,22 @@ public function setThumbAssets(DOMElement $item, array $thumbAssets)
...
 	
+	public function setCaptionAssets (DOMElement $item, array $captionAssets)
+	{
+		foreach ($captionAssets as $captionAsset)
+		{
+			/* @var $captionAsset captionAsset */
+			$content = $this->caption->cloneNode(true);
+			$mediaGroup = $this->xpath->query('media:group', $item)->item(0);
+			$mediaGroup->appendChild($content);
+			$url = $captionAsset->getDownloadUrl(true);
```
Comment (hilak) [inline plugins/content_distribution/providers/comcast_mrss/lib/ComcastMrssFeed.php]: @tan-tan-kanarek  Is there a way to get the captions download URL without the KS? Is there a possibility in your opinion of the KS being cached on the feed beyond the KS expiry date?

Comment (tan-tan-kanarek) [inline plugins/content_distribution/providers/comcast_mrss/lib/ComcastMrssFeed.php]: We can use tokenized URLs just like our serveFlavor actions.

Comment (hilak) [inline plugins/content_distribution/providers/comcast_mrss/lib/ComcastMrssFeed.php]: After checking, I see that the KS expiry is exactly the same as the feed cache expiry - 24 h.

**Comment by tan-tan-kanarek** [comment]
Please add dependency on captions plugin in the comcast plugin.

**Pull request merged**